pkg/target/target.go

```diff
@@ -18,6 +18,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
+// nolint: revive // Moved error out of pkg/webhook/admission; needs capitalization for backwards compat.
+var ErrOldObjectIsNil = errors.New("oldObject cannot be nil for DELETE operations")
```

Comment:
No good place to suggest this change - 

Update [runner_test.go#1160](https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/gator/verify/runner_test.go#L1158) to `{Name: "no oldObject on delete", Error: &clienterrors.ErrorMap{target.Name: constraintclient.ErrReview}},`

And import this in the same test file - 
```
"github.com/open-policy-agent/gatekeeper/v3/pkg/target"
clienterrors "github.com/open-policy-agent/frameworks/constraint/pkg/client/errors"
```

This should fix the CI error.

Reply:
Thank You. I have included those changes.